realm/realm-library/src/androidTest/java/io/realm/RealmMigrationTests.java

```diff
@@ -235,6 +235,84 @@ public void migrate(DynamicRealm realm, long oldVersion, long newVersion) {
         }
     }
 
+    @Test
+    public void removingPrimaryKeyThrows() {
+
+        // Create v0 of the Realm
+        RealmConfiguration originalConfig = configFactory.createConfigurationBuilder()
+                .schema(AllTypes.class)
```

Comment:
This tests will fail after https://github.com/realm/realm-java/pull/3481 fixed.

Maybe simple use DynamicRealm to create only the `StringOnly`?